frontend/src/metabase/visualizations/components/settings/ChartSettingInputNumeric.jsx

```diff
@@ -1,7 +1,19 @@
 /* eslint-disable react/prop-types */
 import React, { Component } from "react";
+import styled from "@emotion/styled";
+import { color } from "metabase/lib/colors";
 
-import cx from "classnames";
+const ChartSettingNumericInput = styled.input`
```

Comment:
We should be trying to use the already existing components because adding a new ones. Unfortunately, we already have multiple different inputs which style is similar to each other. So it is better to avoid adding new ones and use what we have. You may find that the existing component is slightly different in behavior or have slightly different styles — then we should put effort in unifying them

Reply:
As a good example why, this new component has a default outline which is fixed in the core component:
<img width="317" alt="Screen Shot 2022-09-15 at 4 02 29 PM" src="https://user-images.githubusercontent.com/14301985/190398719-60c3ba71-4fbc-4b0a-930c-48f7a1718644.png">

Reply:
Changed to using a `metabase/core/components/Input`, and migrated to TS